com.unity.ml-agents.extensions/Runtime/Sensors/PhysicsSensorSettings.cs

```diff
@@ -40,6 +40,10 @@ public struct PhysicsSensorSettings
         /// </summary>
         public bool UseLocalSpaceLinearVelocity;
 
+        public bool UseJointPositions;
```

Comment:
Not really joint positions. UseJointAngles seems more appropriate to me.

Reply:
"positions" is what ArticulationBody uses (even though it refers to translations or angles). How about `UseJointPositionsAndAngles`?